app/models/concerns/softly_deletable.rb

```diff
@@ -23,6 +23,19 @@ def soft_delete
       end
     end
 
+    def un_soft_delete
+      transaction do
+        before_un_soft_delete if respond_to?(:before_un_soft_delete)
+        result = if is_a?(Claim::BaseClaim)
+                   update_attribute(:deleted_at, nil)
+                 else
+                   update(deleted_at: nil)
```

Comment:
Doesn't both methods `update_attributes` and `update` both update the attribute and saves it value, if so, can we only use one method instead of both?

Reply:
`update_attributes` saves without validation and this is required for claims as it is possible for claims to be 'softly deleted' while in draft state, where they may be invalid.